meetup.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+
+rm _data/meetup/events.json
+curl -o output.json "https://api.meetup.com/openuptown/events?key={YOUR_API_KEY}}"
+cat output.json | jq '.[95:100]' >> _data/meetup/events.json
```

Comment:
This needs to be improved. At this moment there are 100 events returned by the API, what I am trying to do is to return the latest 5. I can either use a variable to store the number of events and simply subtract 5 from it.

I'd like to hear better approaches.

Reply:
This should be pretty easy to remedy. The */events API end point for meetup allows you to use a page argument that takes a number that represents the number of events you want it to respond with. E.g. &page=<some number>

Reply:
Good call.  Let me check it out and I’ll change it.